src/styles/blocks/page.scss

```diff
@@ -0,0 +1,9 @@
+body {
+  justify-content: center;
```

Comment:
Using `justify-content` on the body element requires the body to be a flex container, which is correctly set on line 7. However, ensure that the body has a defined height or width to see the centering effect.